ha/test/integration/health_monitor/test_event_action_handler.py

```diff
@@ -43,13 +43,12 @@ def receive(message):
         event_manager = EventManager.get_instance()
         component = SUBSCRIPTION_LIST.CSM
         resource_type = RESOURCE_TYPES.NODE
-        state = RESOURCE_TYPES.OFFLINE
+        state = RESOURCE_STATUS.FAILED
 
         actions = ["publish"]
         message_type = event_manager.subscribe(component, [SubscribeEvent(resource_type, [state])])
         print(f"Subscribed {component}, message type is {message_type}")
-        health_event = HealthEvent("event_1", "offline", "fault", "site_1", "rack_1", "cluster_1", "storageset_1",
-                                   "node_1", "abcd.com", "node", "16215009572", "disk_1", None)
+        health_event = HealthEvent("source_1", "event_1", "offline", "fault", "site_1", "rack_1", "cluster_1", "storageset_1", "node_1", "abcd.com", "node", "16215009572", "disk_1", None)
```

Comment:
instead of using dummy values here? Can we use proper values? Same comment wherever applicable.

Reply:
using health event values which supports. Fixed.